content/docs/hooks-faq.md

```diff
@@ -455,15 +454,15 @@ Normally, you shouldn't mutate local state in React. However, as an escape hatch
   }
 ```
 
-Try to avoid this pattern if possible.
+Coba hindari pola berikut ini sebisa mungkin.
 
-### Can I make a ref to a function component? {#can-i-make-a-ref-to-a-function-component}
+### Bisakah saya membuat *ref* pada sebuah komponen fungsi? {#can-i-make-a-ref-to-a-function-component}
 
-While you shouldn't need this often, you may expose some imperative methods to a parent component with the [`useImperativeHandle`](/docs/hooks-reference.html#useimperativehandle) Hook.
+Saat Anda seharusnya tidak sering memerlukan hal ini, Anda bisa saja mengekspos beberapa metode perintah pada sebuah *parent component* dengan menggunakan [`useImperativeHandle`](/docs/hooks-reference.html#useimperativehandle) *Hook*.
 
-### How can I measure a DOM node? {#how-can-i-measure-a-dom-node}
+### Bagaimana cara mengukur *DOM node*? {#how-can-i-measure-a-dom-node}
```

Comment:
```suggestion
### Bagaimana cara mengukur sebuah simpul DOM? {#how-can-i-measure-a-dom-node}
```

Reply:
>### Bagaimana cara mengukur *node* DOM? {#how-can-i-measure-a-dom-node}

I keep the 'node' still as 'node', but remove the italicization of DOM.